docs/products/postgresql/howto/use-pg-audit-logging.md

```diff
@@ -0,0 +1,422 @@
+---
+title: Collect audit logs in Aiven for PostgreSQL®
+sidebar_label: Collect audit logs
+pro: true
+---
+
+Enable and configure the [Aiven for PostgreSQL® audit logging feature](/docs/products/postgresql/concepts/pg-audit-logging) on your service. Access and visualize your logs to monitor activities on your databases.
+
+## Prerequisites
+
+- `[Pro Platform](/docs/platform/concepts/pro-platform)`
+- PostgreSQL version 11 or higher
+- `avnadmin` superuser role
+- Dev tool of your choice to interact with the feature
+  - [Aiven Console](https://console.aiven.io/)
+  - [Aiven API](https://api.aiven.io/doc/)
+    ([Aiven API Postman collection](https://www.postman.com/aiven-apis/workspace/aiven/collection/21112408-1f6306ef-982e-49f8-bdae-4d9fdadbd6cd))
+  - [Aiven CLI client](/docs/tools/cli)
+  - [SQL](https://dev.mysql.com/doc/refman/8.3/en/mysql.html)
```

Comment:
This is link points to MySQL. Moreover, I think we should consider using `psql` rather than `SQL` while referring to the standard commandline interface for PostgreSQL.

Here's the link for `psql`: https://www.postgresql.org/docs/current/app-psql.html